### ISSUE TITLE
Props type checking fails with 'module: NodeNext' in tsconfig.json

### ISSUE THREAD

**Issue by ai** [pheuter/svelte-check-rs#4]
`Other.svelte`:

```svelte
<script lang="ts">
    let { binary }: { binary: boolean } = $props();
</script>

{binary ? "1" : "0"}
```

`Test.svelte`

```svelte
<script lang="ts">
    import Other from "./other.svelte";
</script>

<Other wrong />
```

```sh
$ pnpm svelte-check
Loading svelte-check in workspace: /workspaces/svelte-check-rs-test
Getting Svelte diagnostics...

/workspaces/svelte-check-rs-test/web/src/test.svelte:5:8
Error: Object literal may only specify known properties, and '"wrong"' does not exist in type '$$ComponentProps'. (ts)

<Other wrong />

====================================
svelte-check found 1 error and 0 warnings in 1 file

$ ~/.cargo/bin/svelte-check-rs
====================================
svelte-check found 0 errors and 0 warnings in 2 files
```

Reproduction: https://github.com/ai/svelte-check-rs-test/tree/false-negative-prop

**Comment by pheuter** [comment]
@ai can u share your tsconfig.json? here is mine (which correctly generates the errors above)

```json
{
  "extends": "./.svelte-kit/tsconfig.json",
  "compilerOptions": {
    "allowJs": true,
    "checkJs": true,
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "resolveJsonModule": true,
    "skipLibCheck": true,
    "incremental": true,
    "strict": true
  }
}
```

**Comment by ai** [comment]
We can try:

```json
{
  "*.ts": "tsgo",
  "*.svelte": "svelte-check-rs --config web/tsconfig.json"
}
```
